enhancements/baremetal/minimise-baremetal-footprint.md

```diff
@@ -0,0 +1,143 @@
+---
+title: minimise-baremetal-footprint
+authors:
+  - "@hardys"
+reviewers:
+  - "@avishayt"
+  - "@beekhof"
+  - "@crawford"
+  - "@deads2k"
+  - "@dhellmann"
+  - "@hexfusion"
+  - "@mhrivnak"
+approvers:
+  - "@crawford"
+creation-date: "2020-06-04"
+last-updated: "2020-06-05"
+status: implementable
+see-also: compact-clusters
+replaces:
+superseded-by:
+---
+
+# Minimise Baremetal footprint
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Over recent releases OpenShift has improved support for small-footprint
+deployments, in particular with the compact-clusters enhancement which adds
+full support for 3-node clusters where the masters are schedulable.
+
+This is a particularly useful deployment option for baremetal PoC environments,
+where often the amount of physical hardware is limited, but there is still the
+problem of where to run the installer/bootstrap-VM in this environment.
+
+The current solution for IPI baremetal is to require a 4th bootstrap host,
+which is a machine physically connected to the 3 master nodes, that runs
+the installer and/or the bootstrap VM.  This effectively means the minimum
+footprint is 4 nodes, unless you can temporarily connect a provisioning host
+to the cluster machines.
+
+A similar constraint exists for UPI baremetal deployments, where although a
+3 master cluster is possible, you need to run a 4th bootstrap node somewhere
+for the duration of the initial installation.
+
+## Motivation
+
+This proposal outlines a potential approach to avoid the requirement for a
+4th node, leveraging the recent etcd-operator improvements and work to enable
+a live-iso replacement for the bootstrap VM.
```

Comment:
We should reference https://github.com/openshift/enhancements/blob/master/enhancements/rhcos/liveisoinstall.md as this builds on that live-iso installer capability

Reply:
I added this link to the implementation history section as a reference